Source/Core/Core/PowerPC/JitCommon/DivUtils.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <cstdlib>
+
+#include "Core/PowerPC/JitCommon/DivUtils.h"
+
+// Calculate the constants required to optimize a signed 32-bit integer division.
```

Comment:
This description should probably go on the prototype in the header. This way IDE builds with information on mouse hover-over also get the documentation comment.